cpp/src/parquet/chunker_internal_hashtable.h

```diff
@@ -0,0 +1,545 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
If this file is generated, can you:
1) name it `chunked_internal_generated.h`
2) _and_ mark it generated [in the git attributes](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github)

?

Reply:
Renamed, added to git attributes, also renamed the generator file to `chunker_internal_codegen.py`